contracts/mechanics/garden-of-forking-paths/GardenOfForkingPaths.sol

```diff
@@ -16,6 +16,11 @@ import "@openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import "../../diamond/libraries/LibDiamond.sol";
 import "../../diamond/security/DiamondReentrancyGuard.sol";
 
+uint256 constant ERC20_TYPE = 20;
```

Comment:
Maybe we should put this in a library? `LibTokenTypes`?

Reply:
We use it here and in Dropper. If I use these in `StatBlock` as well, I will go ahead and make the library.